perly.y

```diff
@@ -278,14 +278,14 @@ bare_statement_class_declaration
 	:	KW_CLASS
 		BAREWORD[version]
 		BAREWORD[package]
-		subattrlist
+		optattrlist
```

Comment:
May I kindly ask you to use `class_attributes[attributes]` here?

Even if it will be:
```
class_attributes
	:	optattrlist
	;
```

Reasoning:
- there should be name `class` in it
- in future it can be easier to adapt (though this is most likely for me and my effort to switch to GLR parser)
- using renamed pattern may later uncover code duplication and possible branch merges

It also can be later tuned for example into form:
```
class_attributes
	:	{
			push_attribute_context (ATTRIBUTE_CONTEXT_CLASS);
		}
		optattrlist
		{
			pop_attribute_context ();
			$$ = $optattrlist;
		}
	;
```

so code behind `attrlist` can allow only attributes specific for class.